### ISSUE TITLE
Regex language diffs?

### ISSUE THREAD

**Issue by Vlek** [uellenberg/REXS#56]
I like the idea of this project and think that it is helpful to easily and informatively create regular expression statements. My question is whether this is able to deal with the differences between regex languages. .NET, Perl, Python, etc, all have slightly different syntax and available syntax sugar which may or may not make the output of this project work for them.

Have you considered the differences? I could not find anything in the documentation to state what the output should be for.

**Comment by uellenberg** [comment]
No, REXS is built for JavaScript flavored regex. However, making the output match other flavors would only require modifying https://github.com/uellenberg/REXS/blob/master/src/builder.ts, although some flavors have additional features that would be a bit more difficult to implement.
